### PR TITLE
Improve API overview

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Clean up Platform API overview. Includes clearer CTA to view Swagger and download OpenAPI schema.

### CODE REVIEW

**Comment by llewellyn-sl** [comment]
Failing link check is due to updated links not yet merged in https://github.com/seqeralabs/docs/pull/129

**Comment by llewellyn-sl** [comment]
@MythicalFish I tried to fix the position and bottom padding on the buttons. My config changes are having no effect: 

image

File: src/components/Button/index.tsx [hunk 1]
```diff
@@ -1,44 +1,78 @@
...
+                    className
+                )}
+                style={style}
+                role='button'
+                aria-disabled={disabled}
+            >
+                {label}
+            </button>
+        </Link>
     );
```
Comment (ewels) [inline src/components/Button/index.tsx:77]:
```suggestion
    </p>
    );
```

File: src/components/Button/index_old.tsx
[the remark this answers is not in the document]
Comment (llewellyn-sl) [reply src/components/Button/index_old.tsx:1]: Ah, that was just while I was testing updating the TSX to try to improve padding. Forgot to remove.

File: src/components/Button/styles.module.css [hunk 2]
```diff
@@ -1,13 +1,13 @@
 .button {
   display: inline-block;
   color: var(--color-brand-1100);
-  padding: 10px 22px;
+  padding: 20% 10%;
```
[the remark this answers is not in the document]
Comment (llewellyn-sl) [reply src/components/Button/styles.module.css:4]: I tried a couple different things to get padding to work :)

Comment (llewellyn-sl) [inline src/components/Button/styles.module.css:4]:
```suggestion
  padding: 10px 22px;
```

**Comment by llewellyn-sl** [comment]
@ewels that worked, thanks!

**Review by drewdipalma — APPROVED** [review]

**Pull request merged**